### PR TITLE
Add skip-to link for improved accessibility

### PR DESCRIPTION
**Pull request by kylesloper**

### CODE REVIEW

File: www/styles/_base.scss [hunk 1]
```diff
@@ -1,100 +1,127 @@
-@use 'tools' as *;
+@use "tools" as *;
```
Comment (Holben888) [inline www/styles/_base.scss:1]: Noticing some link changes in this file. Luckily, we just merged a fix to add stylelint + a pre-commit hook for enforcement. I'd suggest rebasing onto the latest master and giving `npm run lint` a try

File: www/src/_layouts/docs.njk [hunk 2]
```diff
@@ -20,17 +21,17 @@ styles:
       </a>
     </div>
   </nav>
-  <main class="docs__container">
+  <main class="docs__container" id="main-content">
```
Comment (BenDMyers) [inline www/src/_layouts/docs.njk:24]: Howdy! We should probably add `tabindex="-1"` to this `<main>` to ensure focus moves here in all browsers (including mobile browsers, which tend to struggle with this kind of thing)

More context: <https://benmyers.dev/blog/skip-links/>

Comment (BenDMyers) [reply www/src/_layouts/docs.njk:24]: More context specifically about needing to set the tabindex: https://axesslab.com/skip-links/

**Review by BenDMyers — CHANGES_REQUESTED** [review]

**Review by Holben888 — COMMENTED** [review]
Thanks for this contribution! I agree with adding a skip-link, but we'll need to address Ben and I's (Ben squared?) comments before merging 👍

**Review by BenDMyers — CHANGES_REQUESTED** [review]
Hey, @kylesloper! Thank you so much for putting this together. Skip links could be very helpful for folks using these docs, so thank you for this contribution!

One thing I noticed — on the _homepage_, I think it makes a lot of sense to skip to the `<main>` element. However, on the docs pages, I think `<article class="docs__container__content">` would be a better skip target, since that's the unique contents of the page past all of the navlinks, which people would actually want to skip to.

Would you be willing to make the skip target conditional?

**Comment by kylesloper** [comment]
Sorry for the delayed reply here, I quickly whipped this one up before a surgery I had. In terms of "conditionally" setting the target element, we could do that with some frontmatter magic ✨ but I fear that may be overkill for the docs project.
I have re-defined the aria target for the docs page for now and will commit the new changes.

Let me know if you guys want to go for the conditional setup — it is easily scalable but like I said that might be too much for the docs.

**Comment by Holben888** [comment]
@kylesloper Hm, looks like some conflicts came up here. Double check that you've rebased onto the latest `main` to resolve those. Otherwise I think it'll be good to merge 👍

File: www/src/_layouts/docs.njk [hunk 3]
```diff
@@ -23,17 +24,17 @@ styles:
...
-          {{navLink.title}}
-        </a>
-      </li>
+            {{navLink.title}}
+          </a>
+        </li>
       {%- endfor -%}
     </ul>
-    <article class="docs__container__content">
+    <article class="docs__container__content" id="main-content">
```
Comment (BenDMyers) [inline www/src/_layouts/docs.njk:37]: Howdy- I left a comment a while back that I think got a little lost, but the skip link target should have a `tabindex="-1"` to ensure that, especially in mobile browsers, focus actually moves here.

File: www/src/_layouts/home.njk [hunk 4]
```diff
@@ -25,6 +25,6 @@ styles:
   </div>
 </header>
 
-<main class="home__content">
+<main class="home__content" id="main-content">
```
Comment (BenDMyers) [inline www/src/_layouts/home.njk:28]: This also should have a `tabindex="-1"` to ensure focus is moved in all devices.

**Review by BenDMyers — CHANGES_REQUESTED** [review]

**Review by BenDMyers — APPROVED** [review]

**Comment by kylesloper** [comment]
Pretty sure I have rebased to slinkity's `main` but still seeing conflicts. @Holben888 Any help here :)

**Review by Holben888 — APPROVED** [review]
Won't be skipping this approval!

**Pull request merged**